Makefile

```diff
@@ -71,4 +68,19 @@ exploremem:
 delve:
 	dlv test ./benchmarks -- -test.bench=.
 
-.PHONY: lint test tools install delve exploremem explorecpu profile fullbench bench
+protogen:
+	protoc -I/usr/local/include -I. \
+	-I$(GOPATH)/src \
+	-I$(GOPATH)/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
+	--go_out=plugins=grpc:. \
+	--grpc-gateway_out=logtostderr=true:. \
+	proto/iavl_api.proto
+
+protolint:
```

Comment:
we should use buf or prototool here and also add https://buf.build/docs/tour-5 in ci

Reply:
Seems like tendermint is using protoc too, how about switching both when we learn how to use it?

Reply:
I can switch tendermint to buf. I already have a branch with it but it contains proto breakage